### PR TITLE
order improvements

### PR DESCRIPTION
**Pull request by janiseliang**
* adjust google sheet links based on #71
* change checkbox text (2 checkboxes now)
* some UI improvements
* general code cleanup

### CODE REVIEW

**Comment by janiseliang** [comment]
lmk if theres any UI stuff you think could be improved!

**Comment by justinazhang** [comment]
Yay it looks good!

Just out of curiosity, we don't do this on our end right? Is it done manually by natalia or whoever is managing the orders?
Screen Shot 2022-08-17 at 8 08 58 PM

File: pages/order.js [hunk 1]
```diff
@@ -24,21 +22,19 @@ export default function Order() {
...
 
   // prevent accidentally leaving if past the first page   
-  useEffect(() => {
-    if (formStep > 0) {
-      window.onbeforeunload = () => {
-        return "Leave page? Changes will not be saved.";
-      };
-    }
-  }, []);
+  if (formStep > 0) {
```
Comment (justinazhang) [inline pages/order.js:33]: Woah this is so cool haha

**Review by justinazhang — APPROVED** [review]

**Comment by janiseliang** [comment]
> Just out of curiosity, we don't do this on our end right? Is it done manually by natalia or whoever is managing the orders? Screen Shot 2022-08-17 at 8 08 58 PM

yeah it'll be editable by pantry staff! it's stored in firebase and we'll add it to the admin page in #60

**Pull request merged**